controller_configuration:
  translate_x:
    axis_inc_index: 2
    axis_dec_index: 5 
  translate_y:
    axis_index: 0
  translate_z:
    axis_index: 1
  rotate_roll:
    axis_index: 3
    scale: -2.0
  rotate_pitch:
    axis_index: 4
    scale: -1.0
  rotate_yaw:
    button_inc_index: 0
    button_dec_index: 1
  
